Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,47 @@
+.avatar-circle {
+    max-width: 100%;
+    height: auto;
```

Comment:
same here

Reply:
Thanks for pointing it out.
have made the suitable changes